src/base/emu-i386/simx86/interp.c

```diff
@@ -437,7 +437,7 @@ static unsigned int _Interp86(unsigned int PC, int basemode)
 			}
 		}
 		P0 = PC;	// P0 changes on instruction boundaries
-		if (PC==0 || FetchL(PC)==0 || debug_level('e')) {
+		if (PC==0 || debug_level('e')>1) {
 			e_printf("\n%s\nFetch %08x at %08x mode %x\n",
 				e_print_regs(),FetchL(PC),PC,mode);
```

Comment:
But here the same fetch.
Will it crash with logging enabled?

Reply:
Yes it will, this is just to see if it gets further now for @jschwartzenberg

Reply:
But is this possible to have no
PC fetches elsewhere?